<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWindowTitle</class>
 <widget class="QDialog" name="DialogWindowTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customize Window Title</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Title Definition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select title components to include:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkFileName">
          <property name="text">
           <string>File Name (%f)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkHostName">
          <property name="text">
           <string>Host name (%h)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="checkFileStatus">
            <property name="text">
             <string>File status (%S)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBrief">
            <property name="text">
             <string>brief</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkUserName">
          <property name="text">
           <string>User name (%u)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkClearCase">
          <property name="text">
           <string>Clearcase view tag (%c)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkServerName">
          <property name="text">
           <string>NEdit server name (%s)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkDirectory">
            <property name="text">
             <string>Directory (%d)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelDirectory">
            <property name="text">
             <string>, max components</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editDirectory"/>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>278</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Format:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editFormat">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="editPreview"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Test settings:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkFileModified">
          <property name="text">
           <string>File modified</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkFileReadOnly">
          <property name="text">
           <string>File read only</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkFileLocked">
          <property name="text">
           <string>File locked</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkServerNamePresent">
          <property name="text">
           <string>Server name present</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkClearCasePresent">
          <property name="text">
           <string>CC view tag present</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkServerEqualsCC">
          <property name="text">
           <string>Server name equals CC view tag</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkDirectoryPresent">
          <property name="text">
           <string>Directory present</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogWindowTitle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
